index.html

```diff
@@ -0,0 +1,366 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Scout</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <div class="empty-element none"></div>
+        <div class="header-inner">
+          <a href="#" class="logo">
+            <p>Skout</p>
+          </a>
+          <nav class="container-header-menu none">
+            <ul class="header-menu">
+              <li href="#" class="active-link">WHY SKOUT</li>
```

Comment:
text should be transformed by css
`li` don't have `href` attribute, there must be a link inside